scripts/libs/instance.ts

```diff
@@ -8,6 +8,10 @@ import { executeTx, getFieldFromLogs, getFieldFromTxRcptLogs } from "./transacti
 import { ServiceAddresses } from "./services";
 
 export type InstanceAddresses = {
+    ozAccessManageAddress: AddressLike,
```

Comment:
why is it necessary to expose those 4 addresses? no one else outside of this step needs them. cloning is handled internally in the smart contract.

Reply:
In order to use them in printAddresses()